lib/utils/package-manager.js

```diff
@@ -71,7 +72,34 @@ function getPackageManager() {
 	return "yarn";
 }
 
+/**
+ *
+ * Returns the path to globally installed
+ * npm packages, depending on the available
+ * package manager determined by `getPackageManager`
+ *
+ * @returns {String} path - Path to global node_modules folder
+ */
+function getPathToGlobalPackages() {
+	const manager = getPackageManager();
+
+	if (manager === "yarn") {
+		try {
+			const yarnDir = spawn
+				.sync("yarn", ["global", "dir"])
+				.stdout.toString()
+				.trim();
+			return path.join(yarnDir, "node_modules");
+		} catch (e) {
+			// Default to the global npm path below
+		}
+	}
+
+	return globalPath;
```

Comment:
could we put the require of this into the function? So that in the future no one mixes maybe `globalPath` with `getPathToGlobalPackage()`. 

```
const globalPath = require("global-modules");
const manager = getPackageManager();
```
Then the scope of the function is completely independent.

Reply:
Sure!